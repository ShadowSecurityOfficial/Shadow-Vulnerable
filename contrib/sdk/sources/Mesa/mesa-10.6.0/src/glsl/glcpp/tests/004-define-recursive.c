version https://git-lfs.github.com/spec/v1
oid sha256:8c8e83f52f9aef711eff9ef510eda7c57a5260f7bf750047956658f0885a2d6f
size 60
