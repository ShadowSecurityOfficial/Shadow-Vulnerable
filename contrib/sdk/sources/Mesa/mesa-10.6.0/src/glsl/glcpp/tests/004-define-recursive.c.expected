version https://git-lfs.github.com/spec/v1
oid sha256:41a195565efb9bda730129fa395b585914cad72ec4cd7b05d9656d7c4c9bc712
size 15
